cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -1081,6 +1084,19 @@ func getEnvWithDefaultDuration(k string, defaultVal time.Duration) time.Duration
 	return d
 }
 
+func getEnvWithDefaultBool(k string, defaultVal bool) bool {
+	v := os.Getenv(k)
+	if v == "" {
+		return defaultVal
+	}
+
+	b, err := strconv.ParseBool(v)
+	if err != nil {
+		log.Fatalf("error parsing ENV %s to bool: %s", k, err)
+	}
+	return b
```

Comment:
Ah, yes - that's a mistake on my part .

Reply:
Actually no - this correct (I looked at the full implementation). We only want to return default val if the environment variable is empty (or isn't defined). 

Otherwise, we proceed with trying to parse the environment variable and return the value. We fail outright (in line with the other parsing helpers) if we can't parse the environment variable as a boolean (e.g. it's some non-boolean gobbledygook like "foo")

Reply:
Oooh, `log.Fatalf`. Missed that it would crash the process